galaxy.yml

```
@@ -9,7 +9,7 @@ namespace: azure
 name: azcollection
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 1.13.0
+version: 1.13.1
```

Now we release a version regularly, the next version should be 1.14.0. Thank you!
```suggestion
version: 1.13.0
```